recipe/meta.yaml

```diff
@@ -25,35 +25,37 @@ build:
 requirements:
   host:
     - python
-    - debugpy >=1.0.0
-    - hatchling >=1.4
-    - ipython >=7.23.1
-    - jupyter_client >=6.1.12
-    - packaging
     - pip
-    - comm >=0.1
-    - psutil
+    - setuptools
     - wheel
+    - debugpy
+    - comm
+    - hatchling
+    - ipython
+    - jupyter_client
+    - packaging
+    - psutil
   run:
-    - appnope  # [osx]
-    - comm >=0.1.1
-    - debugpy >=1.0
+    - python
+    - debugpy >=1.6.5
     - ipython >=7.23.1
+    - comm >=0.1.1
+    - traitlets >=5.4.0
     - jupyter_client >=6.1.12
-    - matplotlib-inline >=0.1
     - nest-asyncio
-    - packaging
-    - psutil
-    - python
-    - pyzmq >=17
     - tornado >=6.1
-    - traitlets >=5.4.0
+    - matplotlib-inline >=0.1
+    - appnope  # [osx]
+    - pyzmq >=17
```

Comment:
The upstream pyproject.toml has pyzmq=>20

Reply:
Fixed